app/views/layouts/registration.html.haml

```diff
@@ -10,6 +10,7 @@
       = favicon_link_tag "/favicon-staging.ico"
     %link{href: "https://fonts.googleapis.com/css?family=Roboto:400,300italic,400italic,300,700,700italic|Oswald:300,400,700", rel: "stylesheet", type: "text/css"}
     = render "layouts/matomo_tag"
+    = render "layouts/matomo_tracking"
```

Comment:
The commit message indicates: 
`Adds the tracking matomo code to the darkswarm template`

But it seems that only `app/views/layouts/registration.html.haml` file did change on this commit. May I missed something?

Reply:
Wrong template...Correcting.